crates/anemo-benchmark/build.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+type Result<T> = std::result::Result<T, Box<dyn std::error::Error + Send + Sync>>;
+
+fn main() -> Result<()> {
+    let out_dir = if env::var("DUMP_GENERATED_ANEMO").is_ok() {
```

Comment:
nit: I wonder if we can leave out `DUMP_GENERATED_ANEMO` and just use this behavior when `OUT_DIR` is not set.